Lib/configparser.py

```diff
@@ -687,7 +688,7 @@ def read(self, filenames, encoding=None):
 
         Return list of successfully read files.
         """
-        if isinstance(filenames, str):
+        if isinstance(filenames, (str, bytes, os.PathLike)):
```

Comment:
Right, but those checks were already there when we added `os.PathLike` support to lzma, gzip and other modules. See https://github.com/python/cpython/commit/5f59ddddcd7020bdd3a87b50fe3a8aa0c8e3e689#diff-0949b7e19fdea84affd443e4f7a0c6bfR117 for example. I'd suggest opening another issue to discuss this on bugs.python.org.